src/oic/utils/authn/user.py

```diff
@@ -387,6 +388,10 @@ class SymKeyAuthn(UserAuthnMethod):
 
     def __init__(self, srv, ttl, symkey):
         UserAuthnMethod.__init__(self, srv, ttl)
+
+        if symkey is not None and symkey == "":
```

Comment:
Aha! I see, I see. Thanks for the explanation. I'll follow up with a key length check.